<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryPage</class>
 <widget class="QWidget" name="HistoryPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>854</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>854</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>854</width>
    <height>480</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DINCondensedC</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#HistoryPage{
	border-image: url(:/resource/images/history/historyMap.png);
}




QPushButton{
padding: -1;
}</string>
  </property>
  <widget class="QPushButton" name="button_back">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>76</width>
     <height>100</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">padding: -1;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../../BranQt.qrc">
     <normaloff>:/resource/images/optionPage/button/button_back.png</normaloff>:/resource/images/optionPage/button/button_back.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>76</width>
     <height>100</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_dataName">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>0</y>
     <width>311</width>
     <height>102</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>MI LANTING</family>
     <pointsize>24</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;MI LANTING&quot;;
color:#ffffff</string>
   </property>
   <property name="text">
    <string>PM2.5历史数据</string>
   </property>
  </widget>
  <widget class="QLabel" name="X1">
   <property name="geometry">
    <rect>
     <x>733</x>
     <y>426</y>
     <width>118</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">	font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
   </property>
   <property name="text">
    <string>今天</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="X4">
   <property name="geometry">
    <rect>
     <x>366</x>
     <y>426</y>
     <width>119</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
   </property>
   <property name="text">
    <string>02/11</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="X3">
   <property name="geometry">
    <rect>
     <x>489</x>
     <y>426</y>
     <width>118</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
   </property>
   <property name="text">
    <string>02/12</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="X7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>426</y>
     <width>118</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
   </property>
   <property name="text">
    <string>02/08</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="X6">
   <property name="geometry">
    <rect>
     <x>122</x>
     <y>426</y>
     <width>118</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
   </property>
   <property name="text">
    <string>02/09</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="X2">
   <property name="geometry">
    <rect>
     <x>611</x>
     <y>426</y>
     <width>118</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
   </property>
   <property name="text">
    <string>02/13</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="X5">
   <property name="geometry">
    <rect>
     <x>244</x>
     <y>426</y>
     <width>118</width>
     <height>50</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>118</width>
     <height>0</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
   </property>
   <property name="text">
    <string>02/10</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_hours">
   <property name="geometry">
    <rect>
     <x>558</x>
     <y>100</y>
     <width>274</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>61</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="value_hours">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 60px &quot;KMedium&quot;;
 color:#23a6de;</string>
        </property>
        <property name="text">
         <string>9999</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="unit_2">
        <property name="minimumSize">
         <size>
          <width>71</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>71</width>
          <height>41</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
        </property>
        <property name="text">
         <string>μg/m³</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_days">
   <property name="geometry">
    <rect>
     <x>315</x>
     <y>100</y>
     <width>508</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="value_max">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 60px &quot;KMedium&quot;;
 color:#23a6de;</string>
        </property>
        <property name="text">
         <string>9999</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>5</number>
        </property>
        <property name="bottomMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_max">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
          </property>
          <property name="text">
           <string>最高值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="unit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>25</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
          </property>
          <property name="text">
           <string>μg/m³</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="Line" name="line">
      <property name="minimumSize">
       <size>
        <width>2</width>
        <height>50</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>2</width>
        <height>50</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"> background-color: #A9AAAA; </string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>10</number>
      </property>
      <item>
       <widget class="QLabel" name="value_min">
        <property name="styleSheet">
         <string notr="true">font: 60px &quot;KMedium&quot;;
 color:#29d7c1;</string>
        </property>
        <property name="text">
         <string>9999</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_min">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
          </property>
          <property name="text">
           <string>最低值</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="unit_1">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>200</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font: 20px &quot;MI LANTING&quot;;
    color:#A9AAAA;</string>
          </property>
          <property name="text">
           <string>μg/m³</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QFrame" name="frame_scale">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>410</y>
     <width>854</width>
     <height>12</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>1</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>231</x>
      <y>5</y>
      <width>1</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>728</x>
      <y>5</y>
      <width>1</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>0</y>
      <width>1</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>5</y>
      <width>1</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_0">
    <property name="geometry">
     <rect>
      <x>57</x>
      <y>10</y>
      <width>736</width>
      <height>1</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>368</x>
      <y>5</y>
      <width>1</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>178</x>
      <y>0</y>
      <width>1</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>0</y>
      <width>1</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>0</y>
      <width>1</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>4</y>
      <width>1</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>601</x>
      <y>5</y>
      <width>1</width>
      <height>5</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>418</x>
      <y>0</y>
      <width>1</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_1">
    <property name="geometry">
     <rect>
      <x>792</x>
      <y>0</y>
      <width>1</width>
      <height>10</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"> background-color: #A9AAAA; </string>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../BranQt.qrc"/>
 </resources>
 <connections/>
</ui>
